operations/mimir-mixin/dashboards/slow-queries.libsonnet

```diff
@@ -5,10 +5,154 @@ local filename = 'mimir-slow-queries.json';
+          ],
+          ['p99', 'p50'],
+        )
+      )
+      .addPanel(
+        $.panel('Fetched chunks') +
+        $.lokiMetricsQueryPanel(
+          [
+            'quantile_over_time(0.99, {%s=~"$cluster",%s=~"$namespace",name=~"query-frontend.*"} |= "query stats" != "/api/v1/read" | logfmt | user=~"${tenant_id}" | user_agent=~"${user_agent}" | response_time > ${min_duration} | unwrap fetched_chunk_bytes[$__auto]) by ()' % [$._config.per_cluster_label, $._config.per_namespace_label],
+            'quantile_over_time(0.5, {%s=~"$cluster",%s=~"$namespace",name=~"query-frontend.*"} |= "query stats" != "/api/v1/read" | logfmt | user=~"${tenant_id}" | user_agent=~"${user_agent}" | response_time > ${min_duration} | unwrap fetched_chunk_bytes[$__auto]) by ()' % [$._config.per_cluster_label, $._config.per_namespace_label],
+          ],
+          ['p99', 'p50'],
+          unit='bytes',
+        )
+      )
+      .addPanel(
+        $.panel('Response size') +
+        $.lokiMetricsQueryPanel(
+          [
+            'quantile_over_time(0.99, {%s=~"$cluster",%s=~"$namespace",name=~"query-frontend.*"} |= "query stats" != "/api/v1/read" | logfmt | user=~"${tenant_id}" | user_agent=~"${user_agent}" | response_time > ${min_duration} | unwrap response_size_bytes[$__auto]) by ()' % [$._config.per_cluster_label, $._config.per_namespace_label],
+            'quantile_over_time(0.5, {%s=~"$cluster",%s=~"$namespace",name=~"query-frontend.*"} |= "query stats" != "/api/v1/read" | logfmt | user=~"${tenant_id}" | user_agent=~"${user_agent}" | response_time > ${min_duration} | unwrap response_size_bytes[$__auto]) by ()' % [$._config.per_cluster_label, $._config.per_namespace_label],
+          ],
+          ['p99', 'p50'],
+          unit='bytes',
+        )
+      )
+      .addPanel(
+        $.panel('Time span') +
+        $.lokiMetricsQueryPanel(
+          [
+            'quantile_over_time(0.99, {%s=~"$cluster",%s=~"$namespace",name=~"query-frontend.*"} |= "query stats" != "/api/v1/read" | logfmt | user=~"${tenant_id}" | user_agent=~"${user_agent}" | response_time > ${min_duration} | unwrap duration_seconds(length) [$__auto]) by ()' % [$._config.per_cluster_label, $._config.per_namespace_label],
+            'quantile_over_time(0.5, {%s=~"$cluster",%s=~"$namespace",name=~"query-frontend.*"} |= "query stats" != "/api/v1/read" | logfmt | user=~"${tenant_id}" | user_agent=~"${user_agent}" | response_time > ${min_duration} | unwrap duration_seconds(length) [$__auto]) by ()' % [$._config.per_cluster_label, $._config.per_namespace_label],
+          ],
+          ['p99', 'p50'],
+          unit='s',
+        )
+      )
+    )
+    .addRow(
+      $.row('Top 100 tenants') { collapse: true }
```

Comment:
I think top 100 might be too much. Recently when I've been looking at "top X tenants" 10 was sufficient to find the problematic tenant. WDYT?

Reply:
good call; done in https://github.com/grafana/mimir/pull/6880/commits/c366b9d294a8d47671b583e471ed6f9172e7b86b